components/server/src/ome/services/fulltext/FullTextBridge.java

```diff
@@ -356,6 +362,31 @@ private void handleFileAnnotation(final Document document,
     }
 
     /**
+     * Creates {@link Field} instances for {@link MapAnnotation} named-value
+     * pair.
+     *
+     * @param document
+     * @param store
+     * @param index
+     * @param boost
+     * @param mapAnnotation
+     */
+    private void handleMapAnnotation(final Document document,
+            final LuceneOptions opts, MapAnnotation mapAnnotation) {
+logger().error("handling map annotation");
+        List<NamedValue> nvs = mapAnnotation.getMapValue();
+        if (nvs != null && nvs.size() > 0) {
+            for (NamedValue nv : nvs) {
+                if (nv != null) {
+logger().error("{}={}", nv.getName(), nv.getValue());
+                    add(document, nv.getName(), nv.getValue(), opts);
```

Comment:
I wonder if we want a standard prefix before map keys. (Even also using the map name, if it has a name?) So, they don't collide with other search field names.